app/assets/javascripts/discourse/app/components/sidebar/more-section-links.js

```diff
@@ -0,0 +1,92 @@
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import { bind } from "discourse-common/utils/decorators";
+import GlimmerComponent from "discourse/components/glimmer";
+
+export default class SidebarMoreSectionLinks extends GlimmerComponent {
+  @tracked shouldDisplaySectionLinks = false;
+  @tracked activeSectionLink;
+  @service router;
+
+  constructor() {
+    super(...arguments);
+    this.#setActiveSectionLink();
+    this.router.on("routeDidChange", this, this.#setActiveSectionLink);
+  }
+
+  willDestroy() {
+    this.#removeClickEventListener();
+    this.router.off("routeDidChange", this, this.#setActiveSectionLink);
+  }
+
+  get sectionLinks() {
+    if (this.activeSectionLink) {
+      return this.args.sectionLinks.filter((sectionLink) => {
+        return sectionLink.name !== this.activeSectionLink.name;
+      });
+    } else {
+      return this.args.sectionLinks;
+    }
+  }
+
+  @bind
+  closeDetails(event) {
+    if (this.shouldDisplaySectionLinks) {
+      const isLinkClick = event.target.className.includes(
+        "sidebar-section-link"
+      );
+
+      if (isLinkClick || this.#isOutsideDetailsClick(event)) {
+        document
```

Comment:
I don't see this attribute being added anywhere? Is there a way to do this without manipulating the DOM directly?

Reply:
This attribute part of the `<detail>` element. When it is expanded, the `open` attribute is automatically added to the element.

Reply:
By what though? I can't see it in this PR, only it getting removed

Reply:
The browser will automatically set it.

Reply:
Ohh I see I didn't realise it was like a "special" element, have never used one. Cool :+1:

Reply:
Yea the `<details>` element is special because it is an interactive element. You can read more about the specs [here](https://html.spec.whatwg.org/multipage/interactive-elements.html#the-details-element).